core/templates/pages/story-editor-page/services/story-editor-state.service.ts

```diff
@@ -128,11 +138,15 @@ angular.module('oppia').factory('StoryEditorStateService', [
        * lifetime of the editor. This function never returns null, though it may
        * return an empty story object if the story has not yet been
        * loaded for this editor instance.
+       * @return {Story} Returns the current story to be shared.
```

Comment:
Can we use the text above this instead?
```suggestion
       * @return {Story} The current story to be shared among the story
       * editor. Please note any changes to this story will be propogated
       * ..
       * ..
```

Reply:
Should I in the future just use more of the current description for the params and returns instead of having a description and than add a bit of context to params and returns?

Reply:
Hm, that's a good question. I see both patterns in the codebase currently so I'm not suite sure what needs to be done in this case. Maybe @Showtim3 (the issue owner) can help.